mobile/src/bundles/users/components/image-picker/image-picker.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { PermissionsAndroid } from 'react-native';
+import { type StyleProp, type ViewStyle } from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import { type ImagePickerResponse } from 'react-native-image-picker';
+
+import { Button, Modal } from '~/bundles/common/components/components';
+import { ButtonType } from '~/bundles/common/enums/enums';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks';
+
+type ImagePickerProperties = {
+    label: string;
+    onImageLoad: (payload: Promise<ImagePickerResponse>) => void;
+    containerStyle?: StyleProp<ViewStyle>;
+};
+
+const ImagePicker: React.FC<ImagePickerProperties> = ({
+    label,
+    onImageLoad,
+    containerStyle,
+}) => {
+    const [isPopUpActive, setIsPopUpActive] = useState(false);
+
+    const getImageFromLibrary = useCallback((): void => {
+        setIsPopUpActive(false);
+        const result = launchImageLibrary({ mediaType: 'photo' });
+        onImageLoad(result);
+    }, [onImageLoad]);
+
+    const getImageFromCamera = useCallback((): void => {
+        PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.CAMERA, {
+            title: 'App Camera Permission',
+            message: 'App needs access to your camera ',
+            buttonNeutral: 'Ask Me Later',
+            buttonNegative: 'Cancel',
+            buttonPositive: 'OK',
+        })
```

Comment:
```suggestion
            title: 'Allow BSA Talents to take pictures?',
            buttonNegative: 'Deny',
            buttonPositive: 'Allow',
        })
```
Also on my device, I have multiple dialogs for permissions, this looks incorrect
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/ce1f9be8-4dab-4680-8594-8b1e086f1191)
![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/10cf1d5e-8669-4293-b638-75421f287b88)

Reply:
If I click Deny & don't ask again nothing happens, better add some message about manually allow permission

Reply:
fix